diff -Nprua PyAudio-0.2.14/setup.py PyAudio-0.2.14-LA/setup.py
--- PyAudio-0.2.14/setup.py	2023-11-05 02:08:50.000000000 +0800
+++ PyAudio-0.2.14-LA/setup.py	2024-04-16 10:08:08.332071070 +0800
@@ -72,14 +72,14 @@ def setup_extension():
         # TODO: let users pass in location of portaudio library on command line.
         defines += [('MACOS', '1')]
 
-        include_dirs += [
-            '/usr/local/include', '/usr/include', '/opt/homebrew/include'
-        ]
-        external_libraries_path += [
-            path
-            for path in ('/usr/local/lib', '/usr/lib', '/opt/homebrew/lib')
-            if os.path.exists(path)
-        ]
+        # include_dirs += [
+        #     '/usr/local/include', '/usr/include', '/opt/homebrew/include'
+        # ]
+        # external_libraries_path += [
+        #     path
+        #     for path in ('/usr/local/lib', '/usr/lib', '/opt/homebrew/lib')
+        #     if os.path.exists(path)
+        # ]
 
         if MAC_SYSROOT_PATH:
             extra_compile_args += ["-isysroot", MAC_SYSROOT_PATH]
@@ -111,11 +111,11 @@ def setup_extension():
         # the standard portaudio cmake settings. For details, see:
         # https://devblogs.microsoft.com/cppblog/vcpkg-updates-static-linking-is-now-available/
         extra_compile_args += ["/MT"]
-    else:
+    # else:
         # GNU/Linux and other posix-like OSes will dynamically link to
         # portaudio, installed by the package manager.
-        include_dirs += ['/usr/local/include', '/usr/include']
-        external_libraries_path += ['/usr/local/lib', '/usr/lib']
+        # include_dirs += ['/usr/local/include', '/usr/include']
+        # external_libraries_path += ['/usr/local/lib', '/usr/lib']
 
     return Extension(
         'pyaudio._portaudio',
